### ISSUE TITLE
Linux version does not run

### ISSUE THREAD

**Issue by probonopd** [MayGo/tockler#9]
```
linux@linux:~> /home/linux/Downloads/Tockler-3.0.2-x86_64.AppImage 
6879
{"name":"tockler","hostname":"linux.suse","pid":13345,"level":30,"msg":"Saving logs in directory:/home/linux/.config/Tockler","time":"2017-06-18T19:06:26.202Z","v":0}
{"name":"tockler","hostname":"linux.suse","pid":13345,"logger_name":"AppUpdater","level":30,"msg":"Configuring AutoUpdater.","time":"2017-06-18T19:06:26.270Z","v":0}
A JavaScript error occurred in the main process
Uncaught Exception:
Error: Cannot find module './TrackItemCrud'
    at Module._resolveFilename (module.js:470:15)
    at Function.Module._resolveFilename (/tmp/.mount_Mrp3ar/usr/bin/resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/tmp/.mount_Mrp3ar/usr/bin/resources/app/app/background.service.js:23:23)
    at Object.<anonymous> (/tmp/.mount_Mrp3ar/usr/bin/resources/app/app/background.service.js:486:3)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)

linux@linux:~> cat /etc/*release
NAME="openSUSE Leap"
VERSION="42.2"
ID=opensuse
ID_LIKE="suse"
VERSION_ID="42.2"
PRETTY_NAME="openSUSE Leap 42.2"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:opensuse:leap:42.2"
BUG_REPORT_URL="https://bugs.opensuse.org"
HOME_URL="https://www.opensuse.org/"
```

**Comment by MayGo** [comment]
Working on a fix.

**Comment by MayGo** [comment]
There was problems with camelcase file names. Thats fixed in 3.0.4. Tested on OpenSuse VM.
